### PR TITLE
Update widgets to add replaceable values as key/value pairs

### PR DESCRIPTION
**Pull request by RAVEENSR**
## Purpose
Update widgets to add replaceable values as key/value pairs

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: components/org.wso2.analytics.apim.widgets/APIMApiBackendUsageSummary/src/APIMApiBackendUsageWidget.jsx [hunk 1]
```diff
@@ -213,16 +213,18 @@ class APIMApiBackendUsageWidget extends Widget {
             timeFrom, timeTo, perValue, providerConfig,
         } = this.state;
         const { id } = this.props;
+        const widgetName = this.props.widgetID;
```
Comment (ruwiniwj) [inline components/org.wso2.analytics.apim.widgets/APIMApiBackendUsageSummary/src/APIMApiBackendUsageWidget.jsx]: use destructuring

Comment (ruwiniwj) [reply components/org.wso2.analytics.apim.widgets/APIMApiBackendUsageSummary/src/APIMApiBackendUsageWidget.jsx]: const { id, widgetName } = this.props;

File: components/org.wso2.analytics.apim.widgets/APIMApiCreated/src/APIMApiCreatedWidget.jsx [hunk 2]
```diff
@@ -162,10 +162,12 @@ class APIMApiCreatedWidget extends Widget {
     assembletotalQuery() {
         const { providerConfig } = this.state;
         const { id } = this.props;
+        const widgetName = this.props.widgetID;
```
Comment (ruwiniwj) [inline components/org.wso2.analytics.apim.widgets/APIMApiCreated/src/APIMApiCreatedWidget.jsx]: use destructuring. Fix in other files as well

File: components/org.wso2.analytics.apim.widgets/APIMApiBackendUsageSummary/src/APIMApiBackendUsageWidget.jsx (hunk 1 above)
Comment (RAVEENSR) [reply components/org.wso2.analytics.apim.widgets/APIMApiBackendUsageSummary/src/APIMApiBackendUsageWidget.jsx]: Fixed in e0bba51

File: components/org.wso2.analytics.apim.widgets/APIMApiCreated/src/APIMApiCreatedWidget.jsx (hunk 2 above)
Comment (RAVEENSR) [reply components/org.wso2.analytics.apim.widgets/APIMApiCreated/src/APIMApiCreatedWidget.jsx]: Fixed in e0bba51

File: components/org.wso2.analytics.apim.widgets/APIMApiBackendUsageSummary/src/APIMApiBackendUsageWidget.jsx (hunk 1 above)
Comment (RAVEENSR) [reply components/org.wso2.analytics.apim.widgets/APIMApiBackendUsageSummary/src/APIMApiBackendUsageWidget.jsx]: Fixed in 8fec541

File: components/org.wso2.analytics.apim.widgets/APIMApiCreated/src/APIMApiCreatedWidget.jsx (hunk 2 above)
Comment (RAVEENSR) [reply components/org.wso2.analytics.apim.widgets/APIMApiCreated/src/APIMApiCreatedWidget.jsx]: Fixed in 8fec541

**Pull request merged**